src/Native/GirTestLib/primitivevaluetype.c

```diff
@@ -0,0 +1,38 @@
+#include "primitivevaluetype.h"
```

Comment:
Should the folder: `src/Native/GirTestLib/` be part of the solution to allow easy editing?